[2023-04-30 06:05:36,611] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.sensor_s3_obj manual__2023-04-30T06:05:09.757169+00:00 [queued]>
[2023-04-30 06:05:37,255] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.sensor_s3_obj manual__2023-04-30T06:05:09.757169+00:00 [queued]>
[2023-04-30 06:05:37,257] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-04-30 06:05:37,263] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2023-04-30 06:05:37,272] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-04-30 06:05:38,553] {taskinstance.py:1377} INFO - Executing <Task(S3KeySensor): sensor_s3_obj> on 2023-04-30 06:05:09.757169+00:00
[2023-04-30 06:05:38,691] {standard_task_runner.py:52} INFO - Started process 712 to run task
[2023-04-30 06:05:38,804] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_etl', 'sensor_s3_obj', 'manual__2023-04-30T06:05:09.757169+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_taskflow.py', '--cfg-path', '/tmp/tmpzdzs95fx', '--error-file', '/tmp/tmpmftv4upd']
[2023-04-30 06:05:38,844] {standard_task_runner.py:80} INFO - Job 8: Subtask sensor_s3_obj
[2023-04-30 06:05:40,806] {task_command.py:370} INFO - Running <TaskInstance: dag_etl.sensor_s3_obj manual__2023-04-30T06:05:09.757169+00:00 [running]> on host 295d97741f96
[2023-04-30 06:05:44,440] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Victor
AIRFLOW_CTX_DAG_ID=dag_etl
AIRFLOW_CTX_TASK_ID=sensor_s3_obj
AIRFLOW_CTX_EXECUTION_DATE=2023-04-30T06:05:09.757169+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-30T06:05:09.757169+00:00
[2023-04-30 06:05:44,455] {s3.py:104} INFO - Poking for key : s3://raw/supermarket_sales.csv
[2023-04-30 06:05:45,252] {base.py:68} INFO - Using connection ID 'minio_conn' for task execution.
[2023-04-30 06:05:45,311] {base_aws.py:206} INFO - Credentials retrieved from login
[2023-04-30 06:05:49,167] {base.py:301} INFO - Success criteria met. Exiting.
[2023-04-30 06:05:50,094] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_etl, task_id=sensor_s3_obj, execution_date=20230430T060509, start_date=20230430T060536, end_date=20230430T060550
[2023-04-30 06:05:51,392] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-30 06:05:54,606] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
